Gemfile

```diff
@@ -39,6 +39,8 @@ gem 'rollbar'
 gem 'gibberish'
 gem 'paperclip'
 gem 'bootstrap-sass', '~> 3.2.0'
+gem 'react-rails'
+# gem 'browserify-rails'
```

Comment:
So I'm not sure if I should complicate things using this to load up CommonJS modules and what not for:

1. using other npm modules
2. I need to be able to display assets in the react components (commented below)